src/ImageProcessor.Web.Plugins.PostProcessor/PostProcessor.cs

```diff
@@ -80,6 +78,7 @@ public static async Task<MemoryStream> PostProcessImageAsync(HttpContext context
                             if (File.Exists(destinationFile))
                             {
                                 File.Copy(destinationFile, sourceFile, true);
+                                File.Delete(destinationFile);
```

Comment:
This ensures the process always starts with only a source file, so the processor doesn't need to forcefully overwrite an existing destination file and if it doesn't generate a destination file (like pingo), ensures the processed file is used in the end (when multiple processors are executed).

Reply:
Or should we delete and move/rename, as that would probably cause less disk IOs?

Reply:
The latest changes use `FileInfo` for all IOs and keeps track whether it requires moving the file or not 👍